manifests/system_table.pp

```diff
@@ -1,66 +1,62 @@
 # Add a system table $name to /etc/incron.d
 #
+# If multiple ``path`` and/or ``command`` options are specified, they will be
+# expanded into all matching possibilities.
+#
+# @example Multiplexed Path and Command
+#   incron::system_table { 'test':
+#     path    => ['/foo/bar', '/foo2/bar2'],
+#     command => ['/bin/baz', '/bin/baz2']
```

Comment:
You should have a glob example here if the key part of the new functionality is globbing